src/OrchardCore/OrchardCore.Abstractions/Localization/LocalizationCookieName.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Localization;
+
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Contains a localization cookie names.
+/// </summary>
+public class LocalizationCookieName
+{
+    /// <summary>
+    /// Gets the cookie name that used for the admin site.
+    /// </summary>
+    public static readonly string AdminSite = ".OrchardCore.AdminSiteCulture";
+
+    /// <summary>
+    /// Gets the cookie name that used for the site.
+    /// </summary>
+    public static readonly string Site = CookieRequestCultureProvider.DefaultCookieName;
```

Comment:
=> /// Gets the cookie name that is used for the site culture.

Looks like we are not using this one, if so maybe we don't need this class, only one const but not a big concern.

Reply:
Agree, but my aim it can be used as API from other modules

Reply:
Okay why not ;)